chat-commands.js

```diff
@@ -1762,6 +1762,9 @@ exports.commands = {
 
 		const globalReason = (target ? `: ${userReason} ${proof}` : ``);
 		this.globalModlog((week ? "WEEKLOCK" : "LOCK"), targetUser || userid, ` by ${user.name}${globalReason}`);
+
+		// Automatically upload replays as evidence/reference to the punishment
+		if (room.type === 'battle') this.parse('/savereplay');
```

Comment:
Also this'll need to be `if (room.battle)`

We don't currently support replays of non-battle games.